<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="sim" default="false" />
  <arg name="limited" default="true"/>
  <arg name="debug" default="false" />
    
  <!-- Remap follow_joint_trajectory -->
  <remap if="$(arg sim)" from="/follow_joint_trajectory" to="/arm_controller/follow_joint_trajectory"/>
  
  <!-- Launch moveit (Changed) -->
  <include file="$(find ur5_manipulation)/launch/move_group.launch">
    <arg name="limited" default="$(arg limited)"/>
    <arg name="debug" default="$(arg debug)" />
  </include>
</launch>
